2025-03-21 13:27:22,585 - root - ERROR - Error creating event: (sqlalchemy.dialects.postgresql.asyncpg.Error) <class 'asyncpg.exceptions.DataError'>: invalid input for query argument $3: datetime.datetime(2025, 3, 21, 10, 26, 2... (can't subtract offset-naive and offset-aware datetimes)
[SQL: INSERT INTO events (title, description, start_date, end_date, location, image_url, price, published, created_at, updated_at) VALUES ($1::VARCHAR, $2::VARCHAR, $3::TIMESTAMP WITHOUT TIME ZONE, $4::TIMESTAMP WITHOUT TIME ZONE, $5::VARCHAR, $6::VARCHAR, $7::NUMERIC(20, 8), $8::BOOLEAN, $9::TIMESTAMP WITHOUT TIME ZONE, $10::TIMESTAMP WITHOUT TIME ZONE) RETURNING events.id]
[parameters: ('КИРТАН МЕЛА', 'Киртан мела MM', datetime.datetime(2025, 3, 21, 10, 26, 22, 123000, tzinfo=TzInfo(UTC)), datetime.datetime(2025, 3, 21, 10, 26, 22, 123000, tzinfo=TzInfo(UTC)), 'Куусинена 19а', 'string', 0.0, False, datetime.datetime(2025, 3, 21, 10, 27, 22, 564609), datetime.datetime(2025, 3, 21, 10, 27, 22, 564609))]
(Background on this error at: https://sqlalche.me/e/20/dbapi)
